docusaurus/docs/user-docs/intro.md

```diff
@@ -51,7 +41,7 @@ Clicking on any of the following cards will direct you to the introductory page
 
 <CustomDocCardsWrapper>
   <CustomDocCard emoji="📝" title="Content Manager" description="Manage and publish all content types created with the Content-type Builder." link="/user-docs/content-manager" />
-  <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, Manage and Publish Content Types including collection types." link="/user-docs/content-type-builder" />
+  <CustomDocCard emoji="📚" title="Content Type Builder" description="Create, manage and publish more than one single entries." link="/user-docs/content-type-builder" />
```

Comment:
I would suggest something like the following:

```suggestion
  <CustomDocCard emoji="📚" title="Content Type Builder" description="Build your data structure by creating and managing content-types" link="/user-docs/content-type-builder" />
```

Here's why I'm suggesting this:
- I feel like "publish" is partly wrong: though I can understand what you probably mean, "publishing" something is more related to the Content Manager than to the CTB. 
- I think it's important to keep "content-types" like in your original text, but also to quickly explain what it is.

What do you think? 🙂

Reply:
Hmm How about, "Create and manage content types for multiple entries to build your data structure"?

Reply:
Sounds good too, except maybe for "multiple entries", what do you mean exactly with "multiple"? Because with the CTB you can create collection types (multiple entries) or single types (unique entry). 🤔